config.md

```diff
@@ -8,7 +8,7 @@ mintoclevel = 2
 # uncomment and adjust the following line if the expected base URL of your website is something like [www.thebase.com/yourproject/]
 # please do read the docs on deployment to avoid common issues: https://franklinjl.org/workflow/deploy/#deploying_your_website
 # prepath = "yourproject"
-prepath = "CNRSPostdocSalary"
+prepath = "FrenchPostdocSalary"
```

Comment:
I think we cannot do this, because the website will be deployed at https://adigioacchino.github.io/CNRSPostdocSalary/ and to change this we need to change the website address.